File: AdvantageFramework.Core/Database/Procedures/AlertAttachmentView.cs
using System.Linq;

namespace AdvantageFramework.Core.Database.Procedures
{
    public static partial class AlertAttachmentView
    {
        #region  Constants 



        #endregion

        #region  Enum 



        #endregion

        #region  Variables 



        #endregion

        #region  Properties 



        #endregion

        #region  Methods 

        public static System.Linq.IQueryable<Views.AlertAttachmentView> LoadByAlertID(Database.DbContext DbContext, int AlertID)
        {
            System.Linq.IQueryable<Views.AlertAttachmentView> LoadByAlertIDRet = default;
            LoadByAlertIDRet = from AlertAttachmentView in DbContext.AlertAttachmentViews.AsQueryable()
                               where AlertAttachmentView.AlertId == AlertID
                               select AlertAttachmentView;
            return LoadByAlertIDRet;
        }

        public static System.Linq.IQueryable<Views.AlertAttachmentView> Load(Database.DbContext DbContext)
        {
            System.Linq.IQueryable<Views.AlertAttachmentView> LoadRet = default;
            LoadRet = from AlertAttachmentView in DbContext.AlertAttachmentViews.AsQueryable()
                      select AlertAttachmentView;
            return LoadRet;
        }

        #endregion
    }
}

File: AdvantageFramework/Database/Procedures/AlertAttachmentView.vb
Namespace Database.Procedures.AlertAttachmentView

    <HideModuleName()> _
    Public Module Methods

#Region " Constants "



#End Region

#Region " Enum "



#End Region

#Region " Variables "



#End Region

#Region " Properties "



#End Region

#Region " Methods "

        Public Function LoadByAlertID(ByVal DbContext As Database.DbContext, ByVal AlertID As Integer) As System.Data.Entity.Infrastructure.DbQuery(Of Database.Views.AlertAttachmentView)

            LoadByAlertID = From AlertAttachmentView In DbContext.GetQuery(Of Database.Views.AlertAttachmentView)
                            Where AlertAttachmentView.AlertID = AlertID
                            Select AlertAttachmentView

        End Function
        Public Function Load(ByVal DbContext As Database.DbContext) As System.Data.Entity.Infrastructure.DbQuery(Of Database.Views.AlertAttachmentView)

            Load = From AlertAttachmentView In DbContext.GetQuery(Of Database.Views.AlertAttachmentView)
                   Select AlertAttachmentView

        End Function

#End Region

    End Module

End Namespace
